src/components/Clicking/Clicking.tsx

```diff
@@ -27,8 +27,31 @@ class Clicking extends Component<ClickingProps & ClickingDispatchProps> {
 }
 
 class Pokeball extends Component<{onPokeballClick(): void}> {
+    $pokeball?: HTMLImageElement | null;
+    shakeTimeout?: NodeJS.Timer | null;
+
+    shakePokeball() {
+        if(this.$pokeball == null) {
+            return;
+        }
+        if(this.shakeTimeout != null) {
+            clearTimeout(this.shakeTimeout);
+        }
+        this.$pokeball.classList.add("ShakingBall");
+        this.shakeTimeout = setTimeout(() => this.stopShaking(), 400);
```

Comment:
If you call `window.setTimeout` instead of `setTimeout` TypeScript will be able to find the correct function with a `number` timeout.